crates/corro-types/src/broadcast.rs

```diff
@@ -30,13 +30,22 @@ use crate::{
 #[derive(Debug, Clone, Readable, Writable)]
 pub enum UniPayload {
     V1(UniPayloadV1),
+    V2(UniPayloadV2),
 }
 
+/// Broadcast change payload
 #[derive(Debug, Clone, Readable, Writable)]
 pub enum UniPayloadV1 {
     Broadcast(BroadcastV1),
 }
 
+/// Broadcast change payload that supports priority
+#[derive(Debug, Clone, Readable, Writable)]
+pub enum UniPayloadV2 {
+    Broadcast(BroadcastV1),
+    PriorityBroadcast(BroadcastV1),
+}
```

Comment:
I don't think we need a V2 here. We might even be able to keep a single variant in V1!

I made this little test to check if things could be backwards compatible:

```rust
    #[test]
    fn shenanigs() {
        #[derive(Debug, Readable, Writable)]
        enum A {
            A(Inner),
        }

        #[derive(Debug, Readable, Writable)]
        struct Inner {
            foo: String,
        }

        #[derive(Debug, Readable, Writable)]
        enum B {
            A {
                inner: Inner,
                #[speedy(default_on_eof)]
                flag: bool,
            },
        }

        let v = A::A(Inner { foo: "bar".into() }).write_to_vec().unwrap();

        let b = B::read_from_buffer(&v).unwrap();

        println!("b: {b:?}");

        let v2 = b.write_to_vec().unwrap();
        let a = A::read_from_buffer(&v2).unwrap();

        println!("a: {a:?}");
    }
```